projects/Mallard/src/screens/home-screen.tsx

```diff
@@ -437,15 +440,18 @@ const IssueListFetchContainer = () => {
 
 export const HomeScreen = () => {
     const { issueSummary, error } = useIssueSummary()
-    const {
-        selectedEdition: {
-            header: { title, subTitle },
-        },
-    } = useEditions()
+    const { selectedEdition } = useEditions()
 
+    const specialEditionProps = getSpecialEditionProps(selectedEdition)
     return (
         <WithAppAppearance value={'tertiary'}>
-            <IssuePickerHeader title={title} subTitle={subTitle} />
+            <IssuePickerHeader
+                title={selectedEdition.header.title}
```

Comment:
This looks like a regression to me

Reply:
I'm not sure how to avoid this as the previous notation I think limited the fields on `specialEdition` that were available to just `header` - maybe we can chat about if there's a way of making it work

Reply:
Ah I see! What I was thinking doesnt work. I was expecting the OR in Typescript to give the flexibility I expected but it doesnt.